libsyclinterface/helper/source/dpctl_error_handlers.cpp

```diff
@@ -40,7 +40,7 @@ void DPCTL_AsyncErrorHandler::operator()(
     }
 }
 
-inline int get_requested_level(void)
+inline int requested_verbosity_level(void)
```

Comment:
I added the `inline` without fully understanding it.

Reply:
Yes, if you put it into an anonymous namespace it is analogous to adding `static inline` and it is very likely the compiler will inline it.

Reply:
For some reason CTest executable is not responding to the `DPCTL_VERBOSITY` settings. I suspect the compiler optimizes the `getenv` call away. Is it possible to prevent that?

Reply:
You mean this one?
https://github.com/IntelPython/dpctl/blob/8909c665770a5abcad6c7c40cfc0d001c27c2f73/libsyclinterface/helper/source/dpctl_error_handlers.cpp#L47

Reply:
Yes, I meant this one, but a little debugging session revealed the problem is more mundane. 

libsyclinterface/tests/CMakeLists.txt sets `DPCTL_VERBOSITY=error`, but the test is expecting output issue with level `warning`, and these do not get printed. I am working on fix, along with PR feedback.

Reply:
Right, the compiler will never optimize our the `getenv` cal as it has no way of knowing if the var is set or not. In fact, I have used the trick of adding a dummy `getenv` call to prevent code from getting optimized out.